### PR TITLE
Massively nerfs sneeze.dm [s]

### PR DESCRIPTION
**Pull request by LanCartwright**
:cl: 
tweak:Sneezing now forces you to drop held items.
/:cl:

### CODE REVIEW

**Comment by ChangelingRain** [comment]
kek

**Comment by francinum** [comment]
Nice job on the tag there.

**Comment by Bawhoppen** [comment]
:+1: for the sake of memes
:-1: for adding a changelog

**Comment by GunHog** [comment]
Why would you add the [s] tag here? It is for hiding exploits and security concerns. It is NOT for slipping in nerfs or non-emergency changes.

**Comment by LanCartwright** [comment]
Special Emergancy Request from a headmin directly to me.

**Comment by ChangelingRain** [comment]
It doesn't even work at the end~

**Comment by Supermichael777** [comment]
misleading title

**Comment by Shapsy** [comment]
This just sounds like a massive PITA

**Comment by LanCartwright** [comment]
It concerns me not. I've been given an ultimatum to cut the player problem, or enforce drastic changes to solve this player problem.

**Comment by PKPenguin321** [comment]
if this goes we need more symptoms to up transmission level since i dont think any of them actually do at the moment

also fuck that use of the [s] tag if you're gonna hide your PRs from players at least open your PR between rounds

File: code/datums/diseases/advance/symptoms/sneeze.dm [hunk 1]
```diff
@@ -35,5 +36,13 @@ Bonus
...
-	return
\ No newline at end of file
+				Spreadzone(M, A)
+				var/obj/item/I = M.get_active_hand()
+				if(I)
+					M.drop_item()
+	return
+
+/datum/symptom/sneeze/proc/Spreadzone(mob/living/M, datum/disease/advance/A)
+	var/sneeze_zone = ((sqrt(21+A.totalResistance()))+(sqrt(21+A.totalTransmittable()))/2)
```
Comment (phil235) [inline code/datums/diseases/advance/symptoms/sneeze.dm]: many unneeded parenthesis

Comment (optimumtact) [inline code/datums/diseases/advance/symptoms/sneeze.dm]: MAKE SURE THESE ONES DON'T GO NEGATIVE AS WELL

**Comment by optimumtact** [comment]
Can you just clarify which "headmin" asked you?

Why are you acting all mysterious like david blaine when he fakes his magic tricks.

**Comment by Shadowlight213** [comment]
@optimumtact I'm assuming he meant me?
I asked him to nerf sneezing. Wasn't an emergency special request or anything lol.

File: code/datums/diseases/advance/symptoms/sneeze.dm (hunk 1 above)
Comment (LanCartwright) [inline code/datums/diseases/advance/symptoms/sneeze.dm]: oranges this is an add add divide you cannot go negative on this. That's what the offsets are for.

I'm not sure what you mean by too many Phil. I tried removing a few but I errored out on compiling, and I'm not sure what math system byond uses (PEMDAS or BODMAS).

Comment (phil235) [inline code/datums/diseases/advance/symptoms/sneeze.dm]: `((sqrt(21+A.totalResistance()))+(sqrt(21+A.totalTransmittable()))/2)`

`sqrt(21+A.totalResistance()) + sqrt(21+A.totalTransmittable())*0.5`

**Comment by optimumtact** [comment]
People don't like this, find another way to nerf it.

**Pull request closed without merging**